roles/create-devstack-local-conf/defaults/main.yaml

```diff
@@ -0,0 +1,4 @@
+---
+os_branch: 'master'
+enable_services:
+  - 'default'
```

Comment:
I prefer to show the default service list at here, that's easier to understand, like:

```yaml
enable_services:
  - 'nova'
  - 'neutron'
  - 'cinder'
  - 'keystone'
  - 'glance'
  - 'swift'

Reply:
valuable advice, thanks